src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -294,16 +295,23 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(config : HostConfig<T, P,
         ReactDebugCurrentFiber.current = nextEffect;
       }
 
-      if (nextEffect.effectTag & ContentReset) {
+      const effectTag = nextEffect.effectTag;
+      if (effectTag & ContentReset) {
         config.resetTextContent(nextEffect.stateNode);
       }
 
+      if (effectTag & Ref) {
```

Comment:
I wonder if we should move this check into the switch statement. Already bad that `ContentReset` isn't in there. It requires adding a bunch of combinations (e.g. `PlacementAndRef`, `PlacementAndUpdateAndRef`, `UpdateAndRef`, etc) but it's better for perf.